setup.py

```diff
@@ -3,18 +3,18 @@
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
-REQUIREMENNTS = ["numpy>=1.16.2",
-                 "scipy>=1.0",
-                 "h5py>=2.5.0",
-                 "vtk>=8.1.2",
-                 "tqdm>=4.29.0",
-                 "nibabel>=2.4.0"],
+REQUIREMENTS = ["numpy>=1.24.0",
+                "scipy>=1.10.0",
+                "h5py>=3.7.0",
+                "vtk>=9.2.0",
+                "tqdm>=4.29.0",
+                "nibabel>=5.2.0"],
 
 setuptools.setup(
     name='trimeshpy',
     packages=setuptools.find_packages(),
-    install_requires=REQUIREMENNTS,
-    version='0.0.2',
+    install_requires=REQUIREMENTS,
+    version='0.0.4',
```

Comment:
Simply put back : version='0.0.3'

I will test VTK on my side, and once vtk streamline save is fixed, I will change it to 0.0.4 and push it to pip